language-support/ts/daml-ledger/jest.config.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const moduleNameMapper = {
+  '^@daml/types$': '../daml-types',
+  '^@daml/ledger$': '../daml-ledger',
+  // $1 used for @daml/react/ledgerStore
+  '^@daml/react(.*)$': '../daml-react$1'
```

Comment:
We don't depend on React here. Are you sure we need this?

Reply:
no, this is just copy pasta from the daml-react's jest.config.js. I removed it.